test/definitions/checks/pulpcore/no_running_tasks_test.rb

```diff
@@ -8,14 +8,26 @@
       Checks::Pulpcore::NoRunningTasks.new
     end
 
+    it 'passes when no cli is installed' do
+      assume_feature_present(:pulpcore, :running_tasks => []) do |feature|
+        feature.any_instance.stubs(:cli_available?).returns(false)
+      end
+      result = run_check(subject)
+      assert result.success?, 'Check expected to succeed'
```

Comment:
interestingly, checks that do not pass confinement are still reported as `success`, not `skipped` :woman_shrugging: